client/my-sites/site-settings/section-export.jsx

```diff
@@ -1,88 +1,18 @@
 /**
  * External dependencies
  */
-import React from 'react';
-import Immutable from 'immutable';
+import React, { Component } from 'react';
+import { Provider } from 'react-redux';
 
 /**
  * Internal dependencies
  */
-import CompactCard from 'components/card/compact';
-import Gridicon from 'components/gridicon';
-import Button from 'components/forms/form-button';
-import AdvancedOptions from 'my-sites/exporter/advanced-options';
+import ExporterContainer from 'my-sites/exporter';
 
-const defaults = {
-	advancedSettings: {
-		isVisible: false,
-		posts: {
-			isEnabled: true
-		},
-		pages: {
-			isEnabled: true
-		},
-		feedback: {
-			isEnabled: true
-		}
+export default class SiteSettingsExport extends Component {
+	render() {
+		return React.createElement( Provider, { store: this.props.store }, () => {
+			return <ExporterContainer />;
+		} );
```

Comment:
Because we are using React v0.13, the contained component needs to be wrapped in a function (a known issue). After we go v0.14, it can be written as you described.

However.. this prompted me to think about it some more, and I realized it can just be written this way:

	<Provider store={ this.props.store }>
		{ () => <ExporterContainer /> }
	</Provider>